internal/datanode/flush_manager.go

```diff
@@ -854,6 +854,10 @@ func flushNotifyFunc(dsService *dataSyncService, opts ...retry.Option) notifyMet
 		if pack.flushed || pack.dropped {
 			dsService.channel.segmentFlushed(pack.segmentID)
 		}
+
+		if dsService.flushListener != nil {
+			dsService.flushListener <- pack
```

Comment:
The chan isn't used

Reply:
> The chan isn't used

This can be used for testing in current case.
It can be also used in the future if we want to wait certain flush done.